### PR TITLE
[SERVICES] Homepage 

### PR DESCRIPTION
**Pull request by luizeboli**
## Summary

This PR is part of a bigger one, that references issue #794.

In this one we'll only change the homepage and nothing else, some features still doesn't works because we don't have the API/UI done.

- Search for services
- Services like feature
- Services details card/page (needs definition)
- Services add to cart button

https://www.loom.com/share/1a9a162cac764e77a90d11fd6c5efe9a

## Relevant technical choices

- I chose to write tests when everything's done so we can cover most scenarios
- Did some refactoring, so if in the future we opt to add another "solution" we can easily add it to app flow

## QA Steps

1. Watch Loom video
2. Access app homepage

## Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code passes the linting.
- [x] My code has proper inline documentation.
- [x] I have manually tested this PR.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
     "test:e2e:coverage": "nyc npm run test:e2e:run --reporter=json",
     "test:e2e": "cypress run --headed",
     "merge-coverage": "nyc merge packages/api/.nyc_output .nyc_output/api.json && nyc merge packages/web/coverage .nyc_output/web.json && nyc report",
-    "build-coverage": "npm run coverage && npm run merge-coverage"
+    "build-coverage": "npm run coverage && npm run merge-coverage",
+    "open": "cypress open"
```
Comment (mateus4k) [inline package.json:16]: 🤔

Comment (luizeboli) [reply package.json:16]: OMG

Comment (luizeboli) [reply package.json]: TY

**Review by mateus4k — APPROVED** [review]
Nice feature!

**Pull request merged**